src/main/java/core/org/akaza/openclinica/job/XsltTransformJob.java

```diff
@@ -570,17 +570,28 @@ protected void executeInternal(JobExecutionContext context) {
             postErrorMessage(e.getMessage(), context);
             logger.error("Error executing extract", e);
             exceptions = true;
+        } catch (NullPointerException e) {
```

Comment:
why are you catching NPE? It's usually not a good idea to catch RuntimeExceptions